feat: make the table of status page responsive

utnim2 opened the pull request:
Make the status table mobile responsive

[Screencast from 23-04-24 02:29:13 AM IST.webm](https://github.com/vitessio/arewefastyet/assets/127925465/0a66ad01-27cd-4f88-bcc4-dfe8d6fa83c0)

Related to #500

frouioui reviewed (changes requested):
Thank you for your first contribution! The mobile version looks great to me! However, on a large enough screen the content is not centered properly and was moved to the left:
<img width="1888" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/8a650acb-1222-4a62-98e3-333718291f54">

Moreover, we talked about this offline as you mentioned we should add a third (medium/tablet) size, because right now there is a weird interval when the window is between big and small the content is not responsive at all.

utnim2 commented:
> Thank you for your first contribution! The mobile version looks great to me! However, on a large enough screen the content is not centered properly and was moved to the left: 
>
Fixed in 
> Moreover, we talked about this offline as you mentioned we should add a third (medium/tablet) size, because right now there is a weird interval when the window is between big and small the content is not responsive at all.

fixed in

frouioui commented:
This looks good to me! Thank you for the contribution 🙇🏻
